Dockerfile

```diff
@@ -3,6 +3,7 @@ RUN apt-get update && apt-get install -y --no-install-recommends openssh-client
 RUN go get github.com/mitchellh/gox
 RUN go get github.com/aktau/github-release
 RUN go get github.com/tools/godep
+RUN go get code.google.com/p/go.tools/cmd/cover
```

Comment:
+1 for more coverage unit wise. I wouldn't say we don't care about testing.
It's not apparent but I have integration tests that run against the cli.
It's not perfect but at least 80% of the drivers are regularly tested and
I'm working on getting the remaining in (I need bare metal for those). I'm
wanting to get a way to expose those to the public but right now I can't do
to keys.

This PR is a great step and I think we need it.

Thanks

On Saturday, January 31, 2015, Sebastiaan van Stijn <
notifications@github.com> wrote:

> In Dockerfile
> <https://github.com/docker/machine/pull/453#discussion_r23888367>:
>
> > @@ -3,6 +3,7 @@ RUN apt-get update && apt-get install -y --no-install-recommends openssh-client
> >  RUN go get github.com/mitchellh/gox
> >  RUN go get github.com/aktau/github-release
> >  RUN go get github.com/tools/godep
> > +RUN go get code.google.com/p/go.tools/cmd/cover
>
> If you want to, that may warrant a TODO or FIXME comment?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/docker/machine/pull/453/files#r23888367>.
>